pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -492,6 +492,14 @@ const sourceToPipelineCopyTemplate = `{{- define "sourceToPipelineCopyTemplate"
 
 const pipelineToOutputCopyTemplate = `{{- define "pipelineToOutputCopyTemplate" -}}
 <label {{labelName .Name}}>
+  {{ if .PipelineLabels }}
```

Comment:
it is better to end the template code with an `-`, this doesnt add empty space if there are no labels. e.g.
` {{ if .PipelineLabels -}}`

Reply:
But it doesn't add an empty line even if there are labels. I see that this output:

Reply:
Fixed